### PR TITLE
Create smartloan.toml

### PR DESCRIPTION
**Pull request by Asvoria**
Adding SmartLoan to celo electric capital

### CODE REVIEW

File: data/ecosystems/s/smartloan.toml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+# Sub Ecosystems
+# These are the titles of other ecosystems in different .toml files in the /data/ecosystems directory
+sub_ecosystems = [ "celo" ]
+
+
+
+# Github Organizations
+# This is a list of links to associated github organizations.
+github_organizations = ["https://github.com/celo-org"]
```
Comment (jubos) [inline data/ecosystems/s/smartloan.toml:12]: Can you remove this github organization?

File: data/ecosystems/s/smartloan.toml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Ecosystem Level Information
+title = "SmartLoan"
+
+# Sub Ecosystems
+# These are the titles of other ecosystems in different .toml files in the /data/ecosystems directory
+sub_ecosystems = [ "celo" ]
```
Comment (jubos) [inline data/ecosystems/s/smartloan.toml:6]: Can you capitalize this?

**Review by jubos — CHANGES_REQUESTED** [review]

**Review by Asvoria — COMMENTED** [review]
Done removed the organization and capitalized Celo

**Comment by jubos** [comment]
This is failing the validation.  Can you rebase and fix?

**Comment by jubos** [comment]
I merged your changes into the Celo ecosystem.  We can create a dedicated ecosystem when Smartloans are a bit further along.

**Pull request closed without merging**